pkgs/applications/misc/udevil/default.nix

```diff
@@ -19,11 +27,18 @@ stdenv.mkDerivation {
       --with-setfacl-prog=${acl.bin}/bin/setfacl \
       --sysconfdir=$prefix/etc
   '';
+
+  postInstall = ''
+    substituteInPlace $out/lib/systemd/system/devmon@.service \
+      --replace /usr/bin/devmon "$out/bin/devmon"
+  '';
+
   patches = [ ./device-info-sys-stat.patch ];
-  meta = {
+
+  meta = with lib; {
     description = "A command line Linux program which mounts and unmounts removable devices without a password, shows device info, and monitors device changes";
     homepage = "https://ignorantguru.github.io/udevil/";
-    platforms = lib.platforms.linux;
-    license = lib.licenses.gpl3;
+    platforms = platforms.linux;
+    license = licenses.gpl3;
```

Comment:
Do you know if the license is gpl3Only or gpl3Plus?

Reply:
https://github.com/IgnorantGuru/udevil/blob/ce9f1c5f811e34913b050aed193a39a63ac81e73/COPYING#L640
gpl3Plus

You should make a repo where you list common things to check such as the license